### PR TITLE
Change $Time$ template value for TS segments from uint32 to int64

### PR DESCRIPTION
**Pull request by Canta**
See #701 for details. This PR should fix it.

### CODE REVIEW

**Comment by Canta** [comment]
IDK why it says "10 commits", as I rebased my fork to shaka's latest master, but the PR correctly says "4 files changed" anyways. I guess the commits are a github thing.

I'll leave it to you @kqyang to tell me if you need me to do something else with my fork, or otherwise this is fine.

File: AUTHORS [hunk 1]
```diff
@@ -24,3 +24,4 @@ Piotr Srebrny <srebrny.piotr@gmail.com>
 Richard Eklycke <richard@eklycke.se>
 Sergio Ammirata <sergio@ammirata.net>
 The Chromium Authors <*@chromium.org>
+Daniel Cantarín <canta@canta.com.ar>
```
Comment (kqyang) [inline AUTHORS]: Can you order in alphabetical order, i.e. move it to line 18?

File: CONTRIBUTORS [hunk 2]
```diff
@@ -38,3 +38,4 @@ Richard Eklycke <richard@eklycke.se>
 Rintaro Kuroiwa <rkuroiwa@google.com>
 Sergio Ammirata <sergio@ammirata.net>
 Thomas Inskip <tinskip@google.com>
+Daniel Cantarín <canta@canta.com.ar>
```
Comment (kqyang) [inline CONTRIBUTORS]: ditto

**Comment by kqyang** [comment]
> IDK why it says "10 commits", as I rebased my fork to shaka's latest master, but the PR correctly says "4 files changed" anyways. I guess the commits are a github thing.
> 
> I'll leave it to you @kqyang to tell me if you need me to do something else with my fork, or otherwise this is fine.

Yes, that is fine. We'll do "Squash and merge" so only the changed file matters.

**Comment by Canta** [comment]
For the love of... The files CLEARLY says `Please keep the list sorted`. :man_facepalming: 
I'm so sorry, I was in a hurry with the idea of merging everything today. It's fixed now.

Anyways, I have a question about that. "Contributor" is fine, but... Is it really ok for my name to be in the AUTHORS file? I mean, come on, I'm not any Shaka Packager's "author". I'm just a user who gives something back to the project. I added my name there as per your instructions, and because I guess is the custom/norm for the project, but frankly it feels kinda wrong to be called "author" for this small commits: like it's an important title, but I'm somehow cheating it. 

Nothing serious anyways, just wanted to note it. Do what you consider right to do with the commits.

Thanks!

**Comment by kqyang** [comment]
> Anyways, I have a question about that. "Contributor" is fine, but... Is it really ok for my name to be in the AUTHORS file?

@Canta Yes, we want every contributor to be recognized, regardless of the size of the contribution. AUTHORS is for copyright holders, which is either the author of the commits or their employer.  CONTRIBUTORS is for the author of the commits, so that they are recognized even if they aren't the copyright holder. 

Hope that answers your question. You are welcomed to make more contributions to this project in the future!

**Pull request merged**